README.md

```diff
@@ -1,66 +1,67 @@
 # The OpenAPI Initiative Style Guide
 
-How to (and how not to) refer to the OAI in meetups, interviews, casual conversations, the settling of bar bets, and for conference presentations. 
+How to (and how not to) refer to the **OpenAPI Initiative (OAI)** and the **OpenAPI Specification (OAS)** in meetups, interviews, casual conversations, the settling of bar bets, and for conference presentations. 
 
 ## The Specification Name and Usage
 
-**OpenAPI Specification** refers the name of the popular API description format. It contains ONE space and FIVE capital letters. Following the first instance, it may be referred to as:
+The OAS refers to the name of the popular API description format. It contains ONE space and FIVE capital letters. Following the first instance, it may be referred to as:
 
-* **OAS**: for example, *"Can you send us your OAS document?"*
+| USAGE                 | EXAMPLE                                                  | COMMENT                                   |
+| :-------------------- | :------------------------------------------------------- | :---------------------------------------- |
+| **OAS**               | *Can you send us your OAS document?*                     |                                           |
+| **Specification**<br/>or<br/>**Spec** | *Can you send me your specification?*    |                                           |
+| **OpenAPI**               | *OpenAPI has a different signature mechanism than WSDL.* | Use when comparing to a different format. |
```

Comment:
People still use the word "specification" or "spec" to mean an OpenAPI document, but I thought we were trying to discourage this, so we can reserve "specification" or "spec" to refer to the actual, singular OpenAPI Specification that is maintained in this repository. 

Also, the table makes it sound like "OAS" is to be used only as a modifier (to describe a document or something else), while "OpenAPI" only refers to the format itself, or the specification describing the format. But in fact these two terms are used interchangeably, in both ways, and I don't think there's any problem with that. The meaning is clear by the usage context. 

Suggested changes:

| USAGE                 | EXAMPLES                                                  | COMMENT                                   |
| :-------------------- | :------------------------------------------------------- | :---------------------------------------- |
| **OAS document**<br/>or<br/>**OpenAPI document**              | *Can you send us your OAS document?*<br/><br/>*PoGen creates POJOs from an OpenAPI document.*                     |                                           |
| **OAS**<br/>or<br/>**OpenAPI Specification**<br/>or<br/>**OpenAPI** | *OAS 3.0 organizes reusable definitions into a new `components` object.*<br/><br/>*The OpenAPI Specification allows markdown for formatted API documentation.*<br/><br/>*OpenAPI has a different signature mechanism than WSDL.* | Use when comparing to a different format. |

Reply:
Again, I was just editing and moving the wording into a table for easy reading.  I did not change the previous naming rules.  I, too, found it strange that the previous version allowed referring to the OpenAPI Specification simply as the "specification" or "spec" without requiring any context, but since the rule was there, I did not want to change it.  If the OpenAPI Initiative is trying to discourage this, I can gladly remove it from the table, indicate in the comment that this is being discouraged, or whatever you want.

Reply:
Right, this was an opportunistic comment on my part. Nothing to do with your edits. I just thought, since we're re-working this content, this might be a good time to clarify, update, and improve the recommended terminology. 

@earth2marsh, any comment here...? Am I right to say that we want folks to say "OpenAPI document" or "OAS Document" when they mean an API description in OAS format, and say "OpenAPI Specification" or "spec" when they refer to the standard itself?

Reply:
I will fix it and update my pull request.